editor/editor_style.go

```diff
@@ -51,7 +66,7 @@ type EditorConf struct {
 	ColorScheme string
 }
 
-func NewEditor(editor *Editor, conf *EditorConf, hint string) EditorStyle {
+func NewEditor(editor *Editor, conf *EditorConf, showLineNum bool, hint string) EditorStyle {
```

Comment:
I would suggest to add `showLineNum` in editor config instead if possible.

Reply:
That sounds good to me.